2025-05-19 12:41:04,850 INFO    MainThread:26240 [wandb_init.py:setup_run_log_directory():724] Logging user logs to e:\haselab-all\PyTorch_practice\wandb\run-20250519_124104-uioik67j\logs\debug.log
2025-05-19 12:41:04,850 INFO    MainThread:26240 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to e:\haselab-all\PyTorch_practice\wandb\run-20250519_124104-uioik67j\logs\debug-internal.log
2025-05-19 12:41:04,850 INFO    MainThread:26240 [wandb_init.py:init():852] calling init triggers
2025-05-19 12:41:04,850 INFO    MainThread:26240 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-05-19 12:41:05,859 INFO    MainThread:26240 [wandb_init.py:init():893] starting backend
2025-05-19 12:41:05,860 INFO    MainThread:26240 [wandb_init.py:init():897] sending inform_init request
2025-05-19 12:41:05,860 INFO    MainThread:26240 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-05-19 12:41:05,860 INFO    MainThread:26240 [wandb_init.py:init():907] backend started and connected
2025-05-19 12:41:05,862 INFO    MainThread:26240 [wandb_run.py:_label_probe_notebook():1376] probe notebook
2025-05-19 12:41:05,863 INFO    MainThread:26240 [wandb_init.py:init():1005] updated telemetry
2025-05-19 12:41:05,889 INFO    MainThread:26240 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-05-19 12:41:06,417 INFO    MainThread:26240 [wandb_init.py:init():1104] starting run threads in backend
2025-05-19 12:41:06,529 INFO    MainThread:26240 [wandb_run.py:_console_start():2573] atexit reg
2025-05-19 12:41:06,529 INFO    MainThread:26240 [wandb_run.py:_redirect():2421] redirect: wrap_raw
2025-05-19 12:41:06,529 INFO    MainThread:26240 [wandb_run.py:_redirect():2490] Wrapping output streams.
2025-05-19 12:41:06,530 INFO    MainThread:26240 [wandb_run.py:_redirect():2513] Redirects installed.
2025-05-19 12:41:06,530 INFO    MainThread:26240 [wandb_init.py:init():1150] run started, returning control to user process
2025-05-19 12:41:06,594 INFO    MainThread:26240 [wandb_run.py:_config_callback():1436] config_cb None None {'epochs': 10, 'learning_rate': 0.001, 'batch_size': 256}
2025-05-19 12:42:01,194 INFO    MainThread:26240 [wandb_run.py:_finish():2321] finishing run fukui-university/haselab-conpetition/uioik67j
2025-05-19 12:42:01,194 INFO    MainThread:26240 [wandb_run.py:_atexit_cleanup():2538] got exitcode: 0
2025-05-19 12:42:01,194 INFO    MainThread:26240 [wandb_run.py:_restore():2520] restore
2025-05-19 12:42:01,195 INFO    MainThread:26240 [wandb_run.py:_restore():2526] restore done
2025-05-19 12:42:02,400 INFO    MainThread:26240 [wandb_run.py:_footer_history_summary_info():4188] rendering history
2025-05-19 12:42:02,400 INFO    MainThread:26240 [wandb_run.py:_footer_history_summary_info():4220] rendering summary
2025-05-19 12:42:02,401 INFO    MainThread:26240 [wandb_run.py:_footer_sync_info():4149] logging synced files
